### ISSUE TITLE
NVDA does not announce (all) required group information (Java Access Bridge, panel, groupbox)

### ISSUE THREAD

**Issue by anOolamr** [nvaccess/nvda#13836]
For labeled JPanel(s) of AccessibleRole PANEL, NVDA does not announce the leaving of the group, although this information is transferred via the Java AccessBridge API.
For labeled JPanel(s) of AccessibleRole GROUP_BOX, NVDA does not announce the entering and the leaving of the group, although this information is transferred via the Java AccessBridge API.

I wrote a simple application Test_Group_Announcement.jar and attached it as zip-file to this bug report. (If you prefer a .java file, you can find it within the .jar file.) 
[Test_Group_Announcement.zip](https://github.com/nvaccess/nvda/files/8966362/Test_Group_Announcement.zip)

The jar-file opens a simple frame containing 
- a group of two checkboxes with label "Checkbox group"
- a single checkbox (outside the Checkbox group)
- a group of three radio buttons with label "Radio button group"

The AccessibleJPanel of the JPanel "Checkbox group" returns the AccessibleRole "PANEL".
The (overridden) AccessibleJPanel of the JPanel "Radio button group" returns the AccessibleRole "GROUP_BOX".

The label name of the JPanel is transferred as Parent Name, its AccessibleRole is transferred as Parent Role to the Java AccessBridge API. This can be verified via the jaccessinspector.exe. 

### Steps to reproduce:
Extract the Test_Group_Announcement.jar file from the linked .zip file.

Execuute the attached .jar file with an up-to-date JRE version.
-> A Java window with the title "Frame with two groups" opens.
-> The checkbox with the label "First checkbox" in the "Checkbox group" panel has the keyboard focus.
-> NVDA announces: "Frame with two groups, Checkbox group, panel, First checkbox, checkbox, not checked".
In local language German, NVDA announces "Steuerung" for the "panel" role. This wording is incorrect, as "Steuerung" means "Control" in English. In Java class com.sun.accessibility.internal.resources.accessibility_de the panel key is translated as "Bereich". This would be suitable.

Press Tab key twice.
-> The keyboard focus has left the "Checkbox group" panel. The checkbox "Third checkbox" has the keyboard focus.
-> NVDA announces: "Third checkbox, checkbox, not checked".
-> NVDA does not announce that the "Checkbox group" has been left.

Press Tab key once.
-> The keyboard focus enters the "Radio button group" panel. The radio button "First radio button" has the keyboard focus.
-> NVDA announces: "First radio button, radio button, checked".
-> NVDA does not announce that the "Radio button group" has been entered.

Press keystroke Shift+Tab once.
-> The keyboard focus has left the "Radio button group" panel backwards. The checkbox "Third checkbox" has the keyboard focus.
-> NVDA announces: "Third checkbox, checkbox, not checked".
-> NVDA does not announce that the "Radio button group" has been left.

### Actual behavior:
NVDA does not announce the leaving of labeled JPanel(s) of AccessibleJPanel.AccessibleRole PANEL.
NVDA does not announce the entering and the leaving of labeled JPanel(s) of AccessibleJPanel.AccessibleRole GROUP_BOX.
In local language German, NVDA announces the AccessibleRole PANEL as "Steuerung" (means "control" in English).

### Expected behavior:
NVDA announces the entering and the leaving of labeld JPanel(s).
- AccessibleRole PANEL: "<name> panel", "leaving <name> panel".
- AccessibleRole GROUP_BOX: "<name> group", "leaving <name> group".
In local language German, NVDA announces a JPanel of AccessibleRole PANEL as "Bereich".

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
Version: 2022.1

#### Windows version:
Version: 10.0.19043 Build 19043

#### Name and version of other software in use when reproducing the issue:
java version "18.0.1" 2022-04-19
Java(TM) SE Runtime Environment (build 18.0.1+10-24)
Java HotSpot(TM) 64-Bit Server VM (build 18.0.1+10-24, mixed mode, sharing)

#### Other information about your system:
Java Access Bridge is enabled.

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. Same behavior.

#### If NVDA add-ons are disabled, is your problem still occurring?
Not applicable.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not applicable.

**Comment by Adriani90** [comment]
Just to confirm. In swingset2, the groupings on radio buttons are still not announced with NVDA 2024.4.1. But checkbox groupings are announced properly, so I think NVDA does not support a sort of labelled by something for radio groups.

By the way, eaving a grouping is never announced in Windows applications, see for example MS Office options pannel. In my view this is ok because it is rather important to navigate groupings with ctrl+right and left arrow, and NVDA announces that you entered them. There is no need for extra verbosity when leaving them.

Regarding the translation issue, this needs to be solved by a translator from the german community and this is done via local communication channels (e.g. the german NVDA facebook wor whatsapp group).
